src/test/java/com/wikia/webdriver/common/core/EmailUtils.java

```diff
@@ -20,13 +20,22 @@
 
 import com.wikia.webdriver.common.logging.PageObjectLogging;
 
-public class MailFunctions {
+public class EmailUtils {
 
-  private MailFunctions() {
+  private static final String PASSWORD_RESET_LINK =
+    ".*<a[^>]*href=\"(?<url>[^\"]+?)\"[^>]+>SET NEW PASSWORD</a>.*";
+
+  // Pattern.DOTALL flag forces dot in regex to also match line terminators
+  private static Pattern PASSWORD_RESET_PATTERN =
+    Pattern.compile(PASSWORD_RESET_LINK, Pattern.DOTALL);
+
+  private EmailUtils() {
   }
 
   public static String getFirstEmailContent(String userName, String password, String subject) {
     try {
+      PageObjectLogging.logInfo("Checking emails for " + userName + " that contain '" + subject + "'");
+
       // establishing connections
       Properties props = System.getProperties();
       props.setProperty("mail.store.protocol", "imaps");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "imaps" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)